src/components/Auth/Auth.tsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+
+import InputField from "./InputField/InputField";
+
+export default function Auth() {
```

Comment:
JSX를 보면 form 태그로 시작하는 컴포넌트인데요!
해당 컴포넌트가 하고자 하는 행위 + ui적인 생김새를 드러내주면 좋을 것 같습니다~

저는 우선 이렇게 작성해볼 것 같아요~
```suggestion
export default function SignInForm() {
```